src/test/scala/chiselTests/MultiClockSpec.scala

```diff
@@ -129,6 +129,73 @@ class MultiClockSpec extends ChiselFlatSpec {
     })
   }
 
+  "Differing clocks at memory and port instantiation" should "warn" in {
```

Comment:
The follow 3 cases should be split into differently named tests. This name only applies to the first one, the next 2 are both **not** warnings